nss-day-cohort-48/Rare-API-Spicy-Sauce-Scouts
# added tags

#### Changes Made
1. Added file `tag` to `model/views` directory.
1. Modified file `tag` to include `create, update, delete, retrieve` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cwj-creating-tags
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
